### DISCUSSION TITLE
How to show upcoming events?

### DISCUSSION THREAD

**Discussion by superpower10** [mampfes/hacs_waste_collection_schedule#2094] [Q&A]
I added an custom URL for my waste collection dates in the configuration.yaml. This adds a calendar:

But how do I get this:
![IMG_0920](https://github.com/mampfes/hacs_waste_collection_schedule/assets/20533966/f9eea2d5-540d-48f6-970a-85d485974e87)

```waste_collection_schedule:
  sources:
    - name: ics
      args:
        url: "https://erlangen.mein-abfallkalender.online/ical.ics?xxxxxxxxx
        headers:
          referer: "https://erlangen.mein-abfallkalender.online"
      calendar_title: Abfuhrtermine Stadt Erlangen ```

**Comment by 5ila5** [comment]
It's described in  the [FAQ](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/faq.md) in section "How do I show a coloured Lovelace card depending on the due date?": You need to create a sensor and use a card to display the state of this sensor (built-in or a prettier third party one like  Button Card or [hassio-trash-card](https://github.com/idaho/hassio-trash-card))
